dotcom-rendering/src/components/LightboxLink.tsx

```diff
@@ -0,0 +1,164 @@
+import { css } from '@emotion/react';
+import { ArticleDesign, ArticleDisplay } from '@guardian/libs';
+import {
+	from,
+	neutral,
+	until,
+	visuallyHidden,
+} from '@guardian/source-foundations';
+import { SvgArrowExpand } from '@guardian/source-react-components';
+import type { RoleType } from '../types/content';
+
+type Props = {
+	elementId: string;
+	role: RoleType;
+	format: ArticleFormat;
+	isMainMedia?: boolean;
+	position: number;
+};
+
+function decideSize(role: RoleType, format: ArticleFormat) {
+	const smallStyles = css`
+		height: 32px;
+		width: 32px;
+		svg {
+			height: 20px;
+			width: 20px;
+		}
+	`;
+	const largeStyles = css`
+		height: 44px;
+		width: 44px;
+		svg {
+			height: 32px;
+			width: 32px;
+		}
+	`;
```

Comment:
Is there a reason these are declared inside the function body?

Is it true that for both the `svg` children has a padding of `6px` but otherwise take the whole width?